rootfs-builder/rootfs.sh

```diff
@@ -396,6 +396,9 @@ fi
 
 info "Configure chrony file ${chrony_conf_file}"
 echo "refclock PHC /dev/ptp0 poll 3 dpoll -2 offset 0" >> ${chrony_conf_file}
+# Step the system clock instead of slewing it if the adjustment is larger than
+# one second, at any time
+echo "makestep 1 -1" >> ${chrony_conf_file}
```

Comment:
Idea: this is fine, but now there are two changes to this file, you could instead do:

```sh
cat >>"${chrony_conf_file}"<<EOT
refclock PHC /dev/ptp0 poll 3 dpoll -2 offset 0

# Step the system clock instead of slewing it if the adjustment is larger than
# one second, at any time
￼echo "makestep 1 -1
EOT
```

Reply:
Done